CEDS/MSSQL/0008-View-K12DemographicDim-Create.sql

```diff
@@ -0,0 +1,104 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+IF EXISTS (
+        SELECT 1
+        FROM INFORMATION_SCHEMA.VIEWS
+        WHERE TABLE_SCHEMA = 'xref'
+            AND TABLE_NAME = 'ceds_K12DemographicDim'
+        )
+BEGIN
+    DROP VIEW xref.ceds_K12DemographicDim;
+END;
+GO
+
+CREATE OR ALTER VIEW xref.ceds_K12DemographicDim
+AS
+WITH MapReferenceDescriptor
+AS (
+    SELECT Descriptor.DescriptorId
+        ,Descriptor.CodeValue
+        ,Descriptor.Description
+        ,EdFiTableReference.EdFiTableName
+        ,EdFiTableInformation.EdFactsCode
+    FROM xref.EdFiTableInformation
+    INNER JOIN xref.EdFiTableReference
+        ON EdFiTableInformation.EdFiTableId = EdFiTableReference.EdFiTableId
+    INNER JOIN edfi.Descriptor
+        ON Descriptor.DescriptorId = EdFiTableInformation.EdFiDescriptorId
+    )
+SELECT DISTINCT CONCAT (
+        COALESCE(ReferenceEconomicDisadvantageStatusDescriptor.EdFactsCode, '')
```

Comment:
I think the CONCAT implicitly converts null values to empty strings. So, I don't think we need these 3 COALESCE calls.